splinepy/microstructure/tiles/inversecrosstile3d.py

```diff
@@ -953,12 +953,15 @@ def create_tile(
             )
 
         if closure is not None:
-            return self._closing_tile(
-                parameters=parameters,
-                parameter_sensitivities=parameter_sensitivities,
-                seperator_distance=seperator_distance,
-                closure=closure,
-                **kwargs,
+            return (
```

Comment:
would you like to create a separate mini-PR for this fix?

Reply:
This was changed back into the original layout in `fix-fix-examples`